gluten-substrait/src/main/scala/org/apache/gluten/extension/columnar/validator/Validators.scala

```diff
@@ -137,6 +137,7 @@ object Validators {
       case p: CartesianProductExec if !settings.supportCartesianProductExec() => fail(p)
       case p: TakeOrderedAndProjectExec if !settings.supportColumnarShuffleExec() => fail(p)
       case p: CollectLimitExec if !settings.supportCollectLimitExec() => fail(p)
+      case p: CollectTailExec if !settings.supportCollectTailExec() => fail(p)
```

Comment:
Is this still needed?

Reply:
Validators are for tagging the node with fallback tags. Does the rule `CollectTailTransformerRule` rely on the fallback tags?

Reply:
> this validation would fail for CH and node should be tagged with false.

I see. So you wanted to make it still add a tag but that tag is not actually used for determining whether the node is offload-able. 

This sounds also reasonable. Would be better to have some comments for this on line :140?